CMakeLists.txt

```diff
@@ -297,7 +297,11 @@ if (FMT_PEDANTIC)
   target_compile_options(fmt PRIVATE ${PEDANTIC_COMPILE_FLAGS})
 endif ()
 
-target_compile_features(fmt PUBLIC cxx_std_11)
+if(cxx_std_11 IN_LIST CMAKE_CXX_COMPILE_FEATURES)
+  target_compile_features(fmt PUBLIC cxx_std_11)
+else()
+  message(WARNING "Feature cxx_std_11 is unknown for the CXX compiler")
+endif()
```

Comment:
nit: we use spaces after control flow CMake commands like if/else/endif, e.g.

```
if (...)
```

Reply:
Done